packages/react/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "tsconfig/react-library.json",
-  "include": ["."],
+  "include": [".", "../../vitest.config.ts", "../test"],
```

Comment:
Sometimes these are added by TS automatically, just making sure if these are okay to add here

Reply:
it's just because I don't want strict typing there